src/main/java/de/dennisguse/opentracks/ui/markers/MarkerEditActivity.java

```diff
@@ -121,6 +121,9 @@ protected void onCreate(Bundle savedInstanceState) {
         });
 
         viewModel = new ViewModelProvider(this).get(MarkerEditViewModel.class);
+
+        viewModel.getTrackRecordingServiceConnection().addMarker(getApplication(), null, null,null,null);
```

Comment:
This always creates a new marker even if the user only wants to edit an existing one.

Reply:
I add a if condition